components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2812,6 +2812,13 @@ where
         fail_point!("apply_after_prepare_merge");
         PEER_ADMIN_CMD_COUNTER.prepare_merge.success.inc();
 
+        let range = CacheRange::from_region(&region);
+        info!(
+            "evict range due to prepare merge";
+            "source_range" => ?range,
+        );
+        ctx.engine.evict_range(&range);
```

Comment:
The `evict_range` is not a very fast function which may slow down the apply thread. I think just register the `evict_range` and let it handled by a dedicated worker is better.

Reply:
evict_range should be called immediately. If it is executed at another worker, this delay may be infinitely long.

Reply:
Also, when disable in-memory engine, evict_range will do nothing, so it will have no effect on performance.